dev/tasks/verify-rc/github.linux.amd64.docker.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             -e VERIFY_RC="{{ rc|default("") }}" \
             -e TEST_DEFAULT=0 \
             -e TEST_{{ target|upper }}=1 \
+            -e USE_CONDA=1 \
```

Comment:
We should not add this here. We have `verify-rc-source-java-linux-conda-latest-amd64` for conda. Other jobs such as `verify-rc-source-java-linux-almalinux-8-amd64` should not use conda.

Reply:
Aha, may break CIs and need to figure out a better way here. I am checking.